lib/rubocop/cop/security/open.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Security
+      # This cop checks for the use of `Kernel#open`.
```

Comment:
I'd add more explanations here, as to why this is a security risk. After all this comment is going to become the cop's description in the manual.